lib/babe/epoch.go

```diff
@@ -222,11 +223,25 @@ func (b *Service) incrementEpoch() (uint64, error) {
 // if it is not authorised.
 // output = return[0:32]; proof = return[32:96]
 func (b *Service) runLottery(slot, epoch uint64, epochData *epochData) (*VrfOutputAndProof, error) {
-	return claimPrimarySlot(
+	// TODO: Check if whether to store if a primary slot was claimed? How?
```

Comment:
return a boolean as well?

Reply:
acutally, it doesn't matter here, as this is only used to get the first authoring slot